src/backendbot/api_routes.py

```
@@ -678,3 +539,399 @@ def get_logs(limit: int = 100) -> list[dict[str, Any]]:
+        raise HTTPException(status_code=500, detail=f"Error obteniendo procesos: {e}")
+
+
+@router.post("/kill-process/{pid}", dependencies=[Depends(get_api_key)])
+def kill_process(pid: int):
+    """Mata un proceso por PID."""
+    try:
+        import psutil
+        p = psutil.Process(pid)
+        p.terminate()
+        log_event(f"Proceso {pid} terminado", notify_user=True)
+        return {"status": "ok", "message": f"Proceso {pid} terminado"}
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)